Marlin/language_de.h

```diff
@@ -160,6 +160,8 @@
 #define MSG_LOAD_EEPROM                     _UxGT("Konfig. laden")
 #define MSG_RESTORE_FAILSAFE                _UxGT("Standardwerte laden")
 #define MSG_INIT_EEPROM                     _UxGT("Werkseinstellungen")
+#define MSG_ACTIVATE_SD_UPDATE              _UxGT("SD Update aktivieren")
+#define MSG_DEACTIVATE_SD_UPDATE            _UxGT("SD Update deaktivieren")
```

Comment:
This string is too long. 20 characters is the maximum, and 18 characters or less is preferred.